.github/workflows/podvm_smoketest.yaml

```diff
@@ -0,0 +1,157 @@
+name: Run smoke test on PodVM
+
+on:
+  pull_request:
+
+defaults:
+  run:
+    working-directory: src/cloud-api-adaptor/podvm-mkosi
+
+jobs:
+  smoke-test:
+    runs-on: 'ubuntu-22.04'
+    steps:
+      - uses: actions/checkout@v4
+
+      # Required by rootless mkosi on Ubuntu 24.04
+      # - name: Un-restrict user namespaces
+      #   run: sudo sysctl -w kernel.apparmor_restrict_unprivileged_userns=0
+
```

Comment:
Is this just a reference for developers trying to do this manually, or potentially for future when we can migrate to 24.04 after the OVMF firmware issues are resolved?

Reply:
the latter, because we'd probably forget about this